scripts/convert_to_onnx.py

```diff
@@ -0,0 +1,47 @@
+import torch
+from pathlib import Path
+import argparse
+
+
+def convert_model(model_path: Path, onnx_model_path: Path, opset_version: int):
+    # Restore the model with the trained weights
+    mann_restored = torch.load(str(model_path))
+
+    # Set dropout and batch normalization layers to evaluation mode before running inference
+    mann_restored.eval()
+
+    # Input to the model
+    batch_size = 1
+    x = torch.randn(batch_size, 137, requires_grad=True)
```

Comment:
Let's retrieve the input size from the restored model:

```suggestion
    input_size = next(mann_restored.parameters()).size()[1]
    x = torch.randn(batch_size, input_size, requires_grad=True)
```